app/src/main/java/io/pslab/activity/MainActivity.java

```diff
@@ -52,55 +53,76 @@
 import io.pslab.others.ScienceLabCommon;
 import io.pslab.receivers.USBDetachReceiver;
 
-import static io.pslab.others.ScienceLabCommon.scienceLab;
-
 public class MainActivity extends AppCompatActivity {
 
     private static final String TAG = "MainActivity";
-
+    private static final String TAG_DEVICE = "device";
+    private static final String TAG_INSTRUMENTS = "instruments";
+    private static final String TAG_ABOUTUS = "aboutUs";
+    private static final String TAG_PINLAYOUT = "pinLayout";
+    private static final String TAG_FAQ = "faq";
```

Comment:
There are lot of changes where they are just reshuffling content like this block. Please revert all of them back.